src/components/AgmExtension/ExtensionRequest.vue

```diff
@@ -11,9 +11,12 @@
     <template #content>
       <div
         class="px-6 py-7"
-        :class="{ 'invalid-section': !extensionRequestValid && showErrors }"
+        :class="{ 'invalid-section': !firstSectionValid && showErrors }"
       >
-        <v-row no-gutters>
+        <v-row
+          v-if="!is30MonthsAfterIncorp"
```

Comment:
The UXPin says that if incorporation date is 30 months ago, it is safe to assume that it is not the first agm extension, so hide the radio group and only show the `agmYear` textbox.